src/test/java/io/strimzi/kafka/bridge/http/ConsumerSubscriptionIT.java

```diff
@@ -160,6 +161,92 @@ void subscriptionConsumerDoesNotExistBecauseNotCreated(VertxTestContext context)
         context.completeNow();
     }
 
+    @Test
+    void subscriptionConsumerEmptyTopics(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
+        String name = "my-kafka-consumer-empty-subscription";
+        KafkaFuture<Void> future = adminClientFacade.createTopic(topic);
+        future.get();
+
+        JsonArray topics = new JsonArray();
+        topics.add(topic);
+
+        JsonObject topicsRoot = new JsonObject();
+        topicsRoot.put("topics", topics);
+
+        // Create consumer and subscribe to the topic
+        JsonObject consumerJson = new JsonObject();
+        consumerJson.put("name", name);
+        consumerJson.put("format", "json");
+        consumerService()
+                .createConsumer(context, groupId, consumerJson)
+                .subscribeConsumer(context, groupId, name, topicsRoot);
+
+        // poll to subscribe
+        CompletableFuture<Boolean> consume = new CompletableFuture<>();
+        consumerService()
+                .consumeRecordsRequest(groupId, name, BridgeContentType.KAFKA_JSON_JSON)
+                .as(BodyCodec.jsonArray())
+                .send(ar -> {
+                    if (ar.succeeded()) {
+                        LOGGER.info("Request result: {}", ar.result().body());
+                        consume.complete(true);
+                    }
+                });
+        consume.get(TEST_TIMEOUT, TimeUnit.SECONDS);
+
+        CompletableFuture<Boolean> subscribe = new CompletableFuture<>();
```

Comment:
can you create this later when it is really used just before the subscription to an empty topics list? Maybe even calling `subscribeEmpty` would be better. Thanks!

Reply:
updated, thanks!